### ISSUE TITLE
Problem: hydra-evaluator runs into restrictions

### ISSUE THREAD

**Issue by clacke** [fractalide/tezos-stakepool#14]
```
Oct 02 16:32:50 mail.fractalide.com hydra-evaluator[21368]: starting evaluation of jobset ‘fractalide:tezos-stakepool’ (last checked 600 s ago)
Oct 02 16:33:21 mail.fractalide.com hydra-evaluator[21368]: received jobset event
Oct 02 16:33:22 mail.fractalide.com hydra-evaluator[21368]: hydra-eval-jobs returned exit code 1:
Oct 02 16:33:22 mail.fractalide.com hydra-evaluator[21368]: error: access to path '/nix/store/9445svhcspwaadrm26v2raclw3x4ayc9-q9nhs2f1lpzhmy6knagrahnyviypnvgr-git-export/nixos' is forbidden in restricted mode
Oct 02 16:33:22 mail.fractalide.com hydra-evaluator[21368]: evaluation of jobset ‘fractalide:tezos-stakepool’ failed with exit code 1
```

**Comment by clacke** [comment]
Probably due to #8 ? But I can't figure out how it would get hold of a reference that wouldn't be in its `NIX_PATH`.

**Comment by clacke** [comment]
Ah, no! 20b5419 built just fine. It's the pinning that makes it fail, and I think I understand why.

We get the pinned nixpkgs via a derivation. Fine. But then we point out `nixos` inside that derivation through path manipulation and that's naughty of us. So there needs to be some better way to do that.

**Comment by clacke** [comment]
Lots of discussion in NixOS/nix#954, but I'm not entirely clear on exactly why it doesn't work this time, but works for the pinnings we have in other repos. I'm still guessing it's the path manipulation, but I'm not sure how to work around that. There doesn't seem to be a way to access `nixpkgs/nixos` via attributes in `nixpkgs`.

**Comment by clacke** [comment]
No, hang on. It's not saying `/nix/store/7hd1al7czby6268y14w6w9n8dys4g17y-nixpkgs-channels-c1a66a4/nixos`, which would be the pinned repo. This `...-git-export` is one of the repos hydra checked out. So now I don't get again why we're restricted from using it, because it should be on the `NIX_PATH` of the evaluator.

**Comment by clacke** [comment]
Closed by #15 .
